.github/workflows/build-and-test.yml

```diff
@@ -58,8 +60,29 @@ jobs:
           key: go-pkg-mod-${{ runner.os }}-${{ hashFiles('./go.mod') }}
       - name: Install tools
         run: make install-tools
+      - name: split loadtest jobs
+        id: splitloadtest
+        run: |
+          TESTS="$(make -s testbed-list-loadtest | xargs echo|sed 's/ /|/g')"
+          TESTS=(${TESTS//|/ })
+          MATRIX="{\"include\":["
+          curr=""
+          for i in "${!TESTS[@]}"; do
+            if (( i > 0 && i % 2 == 0 )); then
+              curr+="|${TESTS[$i]}"
+            else
+              if [ -n "$curr" ] && (( i>1 )); then
+                MATRIX+=",{\"test\":\"$curr\"}"
+              elif [ -n "$curr" ]; then
+                MATRIX+="{\"test\":\"$curr\"}"
+              fi
+              curr="${TESTS[$i]}"
+            fi
+          done
+          MATRIX+="]}"
+          echo "::set-output name=matrix::$MATRIX"
       - name: Upload tool binaries
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v1
```

Comment:
Turns out the flakiness of err CONNRESET when download/uploading artifacts is a known bug in `upload-artifact@v2` as noted [here](https://github.com/actions/upload-artifact/issues/71). reverting to `@v1` for now